httplib.h

```diff
@@ -4409,8 +4412,69 @@ inline bool load_system_certs_on_windows(X509_STORE *store) {
 
   return true;
 }
+#elif defined(__APPLE__)
+template <typename T>
+using CFObjectPtr = std::unique_ptr<T, void (*)(CFTypeRef)>;
```

Comment:
Can you move `CFObjectPtr` into `load_system_certs_on_apple`?

Reply:
It is template alias. It is not allowed to declare a template inside a function